curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-96-su-doku.english.md

```diff
@@ -7,6 +7,7 @@ forumTopicId: 302213
 
 ## Description
 <section id='description'>
+<<<<<<< HEAD
```

Comment:
Merge stuff

Reply:
Sorry about these. Not sure why my editor didn't pick up on them. I'll get these all worked out.